src/aws_s3_auth_conn.tsx

```diff
@@ -38,15 +56,20 @@ export class MyS3Auth {
     this.validUser = false
```

Comment:
User validity shouldn't be false after changing bucket name right

Reply:
I think it should remain false because the user may not have access to another bucket that they've entered, or they entered the wrong bucket name so we can't automatically assume the user is still valid.